### PR TITLE
Seamless posting

### PR DESCRIPTION
**Pull request by SummerWish**

### CODE REVIEW

File: Help/content_server.html [hunk 1]
```diff
@@ -104,6 +107,36 @@
...
+         <b class="caret">
+         </b>
+         Options
+        </a>
+        <ul class="dropdown-menu">
+         <li>
+          <a class="account_url logged_in_nav" href="#" style="display:none;" target="_blank">
+           <span class="glyphicon glyphicon-new-window">
+           </span>
+           Server Account
```
Comment (smcgregor) [inline Help/content_server.html]: You should build `python build.py --platform=web` for the develop and master branches. The Chrome-specific build goes in chrome-extension-master.

File: Message/embeded_ttlselect.html.subtemplate [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{% extends "templates/embeded_ttlselect.html.template" %}
+
+{% block javascripts %}
+  <script type="text/javascript" src="js/message.js"></script>
+  <script type="text/javascript" src="js/embeded_ttlselect.js"></script>
+{% endblock %}
```
Comment (smcgregor) [inline Message/embeded_ttlselect.html.subtemplate]: nit: need newlines at the end of a lot of these.

**Comment by smcgregor** [comment]
Do you have this building on TravisCI?

**Comment by smcgregor** [comment]
I think we are going to need to break this pull request into several parts if we are going to be able to discuss it all and make changes, particularly as we look to integrate the changes with JetPack and Safari.

Maybe break the changes into separate pull requests for:

1. Miscellaneous changes (things you noticed and fixed)
1. Updating current HTML templates
1. Rendering of current HTML pages
1. Changes to current libraries that are needed for the seamless posting
1. Changes to the Message app that helped integrate seamless posting.
1. The new scripts and templates for seamless posting.
1. Rendering of new HTML pages

**Comment by SummerWish** [comment]
See #274

**Pull request closed without merging**